src/common/example/index.html

```diff
@@ -38,25 +38,7 @@
 			}
 		}));
 
-		(function () {
-			var query = window.location.search.substring(1).split('&');
```

Comment:
Can we make this continue to work with a query string? Or is there a workaround I'm not thinking of to run the functional tests?

Reply:
I didn't realise the functionals used the same index.html file as the examples... we'd need to resolve this.